views/parsely-settings.php

```diff
@@ -44,6 +44,14 @@
 		<?php
 		settings_fields( Parsely::OPTIONS_KEY );
 		$wp_parsely_settings->show_setting_tabs_content();
+
+		// phpcs:ignore WordPress.Security.NonceVerification.Recommended
+		if ( isset( $_GET['e2e_parsely_skip_api_validate'] ) && 'y' === $_GET['e2e_parsely_skip_api_validate'] ) {
+			?>
+			<input type="hidden" name="e2e_parsely_skip_api_validate" value="y" />
```

Comment:
Question: Does this field risk of being saved into the `parsely` options array?

Reply:
Should be safe since it isn't using the Settings API. But good question, I'm going to test that and make sure it isn't leaking to the options array.

Reply:
Confirmed: it's not leaking the field into the options array.

Reply:
Great, thank for checking! 👍